### ISSUE TITLE
optifine profile broke

### ISSUE THREAD

**Issue by CosmicRedWolfRx** [SpacePython12/AP-Launcher#3]
idk if there is anything you can do but when I downloaded optifine it wouldn't launch my game.
***
Log contents:
```
[2021-10-25 18:37:29,721] (Process MainProcess thread MainThread func <module>/INFO): Version is modified, inheriting from 1.17.1.
[2021-10-25 18:37:29,721] (Process MainProcess thread MainThread func <module>/ERROR): 'jvm'
Traceback (most recent call last):
  File "launcher_process.py", line 218, in <module>
KeyError: 'jvm'

```

**Comment by SpacePython12** [comment]
I'll work on it, should be a simple fix.

**Comment by SpacePython12** [comment]
Fixed in version 0.11.
